docs/partner_editable/faq_troubleshooting.adoc

```diff
@@ -18,4 +18,13 @@ For additional information, see https://docs.aws.amazon.com/AWSCloudFormation/la
 
 //== Troubleshooting
 
-//<Steps for troubleshooting the deployment go here.>
\ No newline at end of file
+//<Steps for troubleshooting the deployment go here.>
+
+
+//TODO Shivansh, Follow this FAQ, do you know why the guide includes only the parameters for launching into a new VPC?
```

Comment:
Seems like somehow the entrypoint for existing VPC got deleted and therefore the param table is not being generated. I'll fix it in my branch and push the changes with other edits.

Reply:
Okay.